src/shapes/ply.cpp

```diff
@@ -111,28 +112,28 @@ class PLYMesh final : public Mesh<Float, Spectrum> {
             fail(e.what());
         }
 
-        // TODO check header float type (32 vs 64)
-
         bool has_vertex_normals = false;
+        bool has_vertex_texcoords = false;
+
+        ref<Struct> vertex_struct;
+        ref<Struct> face_struct;
```

Comment:
Those two could probably be allocated on the stack.

Reply:
Indeed! Thanks